src/java/org/apache/cassandra/db/marshal/ListType.java

```diff
@@ -76,6 +77,17 @@ private ListType(AbstractType<T> elements, boolean isMultiCell)
         this.isMultiCell = isMultiCell;
     }
 
+    @Override
+    public ListType<T> overrideKeyspace(Function<String, String> overrideKeyspace)
+    {
+        List<AbstractType<?>> subTypes = subTypes();
+        AbstractType<?> newType = subTypes.get(0).overrideKeyspace(overrideKeyspace);
+        if (newType == subTypes.get(0))
+            return this;
+
+        return new ListType<>((AbstractType<T>) newType, isMultiCell());
```

Comment:
see https://github.com/datastax/cassandra/pull/331/files#r801303769, we should use getInstance to reduce GC.